utils/setup.cfg

```diff
@@ -7,10 +7,13 @@ universal=1
 [nosetests]
 tests=../,../roles/openshift_master_facts/test/,test/
 verbosity=2
-with_coverage=1
-cover_html=1
-cover_package=ooinstall
-cover_min_percentage=70
+with-coverage=1
+cover-html=1
+#cover-package=ooinstall,modify_yaml,openshift_master_facts_default_predicates,openshift_master_facts_default_priorities
```

Comment:
Just something I accidentally left in after testing. Will remove shortly, wanted to wait for testing to complete before I removed it (I noticed after I tagged for testing).